04_009/server/src/app.ts

```diff
@@ -0,0 +1,38 @@
+import express, { Application } from 'express';
+import errorMiddleware from '@/middlewares/error-middleware';
+import loaders from './loaders';
+import router from './routes';
+import CustomError from './error/custom-error';
+import { NOT_FOUND_ERROR } from './constants/error';
+
+class App {
+  app: Application;
+
+  constructor() {
+    this.app = express();
+    this.setLoaders();
+    this.setRouter();
+    this.setNotFoundHandler();
+    this.setErrorMiddleware();
+  }
+
+  async setLoaders() {
+    await loaders(this.app);
+  }
```

Comment:
```suggestion
  setLoaders() {
    loaders(this.app);
  }
```
async/await은 지금 없어도 무방합니다!